doc/index.rst

```diff
@@ -6,40 +6,55 @@
 lsst.ts.MTDomeTrajectory
 ########################
 
-ts_MTDomeTrajectory contains the `MTDomeTrajectory` CSC and support code,
-including a simple `MockDome` CSC for unit testing.
+.. image:: https://img.shields.io/badge/SAL\ Interface-gray.svg
+    :target: https://ts-xml.lsst.io/sal_interfaces/MTDomeTrajectory.html
+.. image:: https://img.shields.io/badge/GitHub-gray.svg
+    :target: https://github.com/lsst-ts/ts_MTDomeTrajectory
+.. image:: https://img.shields.io/badge/Jira-gray.svg
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_MTDomeTrajectory
 
-.. _lsst.ts.MTDomeTrajectory-using:
+Overview
+========
 
-Using lsst.ts.MTDomeTrajectory
-==============================
+MTDomeTrajectory commands the Simonyi Survey Telescope dome to follow the telescope.
```

Comment:
...follow the telescope based on a user-defined trajectory.
Unlike most observatory enclosures, this dome "crawls" during exposures to minimize the acceleration and therefore time required to move to the next target.

Reply:
It is expected the algorithm to minimize dome repositioning times will evolve with time and conditions, therefore multiple trajectory profiles are supported.

Reply:
How about this:
```
MTDomeTrajectory commands the Simonyi Survey Telescope dome to follow the telescope.
It does this by reading telescope position from the MTMount CSC and issuing commands to the Dome CSC.

Unlike most observatory enclosures, this dome slowly rotates during exposures, in order to minimize the time required to move to the next target.
MTDomeTrajectory supports multiple algorithms for determining how to most efficiently move the dome, though for now only one simple algorithm has been implemented.
```
I am reluctant to use the term "trajectory" because it appears nowhere in the code (though I realize it appears in the CSC name).